src/main/java/edu/cornell/kfs/concur/service/impl/ConcurAccountValidationServiceImpl.java

```diff
@@ -92,7 +95,8 @@ public ValidationResult checkRequiredAccountInfo(String chartOfAccountsCode, Str
 
     public ValidationResult checkAccount(String chartOfAccountsCode, String accountNumber) {
         Account account = accountService.getByPrimaryId(chartOfAccountsCode, accountNumber);
-        return checkMissingOrInactive(account, MessageFormat.format(configurationService.getPropertyValueAsString(KFSKeyConstants.ERROR_EXISTENCE), ConcurConstants.AccountingStringFieldNames.ACCOUNT_NUMBER), MessageFormat.format(configurationService.getPropertyValueAsString(KFSKeyConstants.ERROR_INACTIVE), ConcurConstants.AccountingStringFieldNames.ACCOUNT_NUMBER));
+        String  accountErrorMessageString = ConcurConstants.AccountingStringFieldNames.ACCOUNT_NUMBER + CUKFSConstants.COLON + chartOfAccountsCode + KFSConstants.COMMA + accountNumber;
```

Comment:
there are a few cases where you are building an errorMessageString using a similar format: fieldName:value,value .  Would it be helpful to break that into a function?  I am asking about this.